### ISSUE TITLE
Resolving conflict while releasing may cause early neptune termination

### ISSUE THREAD

**Issue by mihails-strasuns-sociomantic** [sociomantic-tsunami/neptune#195]
After resolving conflicts I did bunch of local testing but neptune crashed after existing conflict resolution shell:

```
lib.shell.helper.ExitCodeException@./src/lib/shell/helper.d(79): Cmd Failed: error: Your local changes to the following files would be overwritten by checkout:
	src/ocean/LibFeatures.d
	src/ocean/core/SmartUnion.d
	src/ocean/core/UnitTestRunner.d
	src/ocean/math/random/Kiss.d
	src/ocean/math/random/Random.d
        ...
```

This has left repository in half-finished state and I had to finish release process by hand :)

**Comment by mathias-baumann-sociomantic** [comment]
Can you describe this in more detail? Did you successfully resolve the conflict? Is there maybe more log available?

**Comment by mathias-baumann-sociomantic** [comment]
Ah, I understand now. Yeah, so far I trust that the user left the repository in a state where I can operate on it normally. Unstashed changes are simply not accounted for. I suppose I will need to add a check after the shell returns to see if the git repo is in a clean state.

**Comment by mihails-strasuns-sociomantic** [comment]
Yup, just checking if it is in clean state and returning back to shell with a request to clean up before proceeding would be ideal.
